j/jackson-modules-java8/build-info.json

```diff
@@ -0,0 +1,14 @@
+{
+    "package_name" : "jackson-modules-java8",
+    "github_url":"https://github.com/FasterXML/jackson-modules-java8",
+    "version": "jackson-modules-java8-2.14.1",
+    "package_dir": "j/jackson-modules-java8",
+    "build_script": "jackson_module_java8_ubi_8_5.sh",
+    "validate_build_script": "true",
+    "use_non_root_user": "true",
```

Comment:
If `use_non_root_user` is set to`true`, build-script will be validated as non root used (sudo user).
But as per [this line](https://github.com/ppc64le/build-scripts/blob/89d614ba2c526b1ad51ad85a06e5facfd316bd94/j/jackson-modules-java8/jackson_module_java8_ubi_8_5.sh#L32), you are installing packages (using yum) without sudo.

I doubt, this would be gives troubles while validation. Please check and update accordingly.

If your intention to run it as non root user, please update the build-script (add sudo, where ever you need extended privileges) also update `Disclaimer` part of the script (Mention, script is tested as non-root user in Disclaimer).

Reply:
In that case it would be good if I make it "False(default-value)" as mine intension is not to run it as non root user.